docs/EN_US/ECLWatch/TheECLWatchMan.xml

```diff
@@ -418,26 +418,40 @@
 
           <para>The <emphasis role="bold">Set Banner</emphasis> link allows
           you to set a custom banner message at the top of your browser window
-          when you open ECL Watch. You can use this feature to send messages
-          to users.</para>
+          when you open ECL Watch. You check the <emphasis
+          role="bold">Enable</emphasis> box to enable your banner. You can
+          then use this feature to send messages to users and you can
+          customize the appearance of that message with the other controls
+          there. Banner messages persist until restarts.</para>
+
+          <para>The <emphasis role="bold">Set Toolbar</emphasis> link allows
```

Comment:
The Set Toolbar link allows you to customize the toolbar at the top of the ECL Watch page. 
Check the Enable Environment Text box to display the Name of Environment text at the top of the page and in the browser tab. Labeling browser tabs is helpful when working with multiple environments. Toolbar settings persist through restarts of the ESP Server.